src/components/Autocomplete.tsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
```

Comment:
try to get rid of it
some info: 
https://dev.to/shivampawar/error-non-interactive-elements-should-not-be-assigned-mouse-or-keyboard-event-jsx-a11yno-noninteractive-elements-interactions-2i1g

https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-noninteractive-element-interactions.md